sections/futures/Trades/Trades.tsx

```diff
@@ -52,133 +76,158 @@ const Trades: React.FC<TradesProps> = ({ history, isLoading, isLoaded }) => {
-									{cellProps.row.original.side}
-								</LeverageSide>
+								<StyledPositionSize>{formatCryptoCurrency(cellProps.value)}</StyledPositionSize>
 							</FlexDivCentered>
 						),
 						width: 100,
 						sortable: true,
 					},
+					// {
+					// 	Header: (
+					// 		<StyledTableHeader>
+					// 			{t('futures.market.user.trades.table.trade-type')}
+					// 		</StyledTableHeader>
+					// 	),
+					// 	accessor: 'type',
+					// 	sortType: 'basic',
+					// 	Cell: (cellProps: CellProps<FuturesTrade>) => (
+					// 		<FlexDivCentered>
+					// 			{/* <CurrencyIcon currencyKey={cellProps.row.original.asset ?? ''} /> */}
+					// 			<StyledPositionSize>{cellProps.value}</StyledPositionSize>
+					// 		</FlexDivCentered>
+					// 	),
+					// 	width: 100,
+					// 	sortable: true,
 					{
 						Header: (
-							<StyledTableHeader>{t('futures.market.user.trades.table.entry')}</StyledTableHeader>
+							<StyledTableHeader>{t('futures.market.user.trades.table.price')}</StyledTableHeader>
 						),
-						accessor: 'entryPrice',
+						accessor: 'value',
 						sortType: 'basic',
-						Cell: (cellProps: CellProps<PositionHistory>) => (
-							<Price>
-								{formatCurrency(Synths.sUSD, cellProps.value, {
-									sign: '$',
-								})}
-							</Price>
+						Cell: (cellProps: CellProps<FuturesTrade>) => (
+							<>
+								<Price>
+									{formatCurrency(Synths.sUSD, cellProps.value, {
+										sign: '$',
+									})}
+								</Price>
+							</>
 						),
 						width: 100,
 						sortable: true,
 					},
+					// {
+					// 	Header: (
+					// 		<StyledTableHeader>{t('futures.market.user.trades.table.final')}</StyledTableHeader>
+					// 	),
+					// 	accessor: 'exitPrice',
+					// 	sortType: 'basic',
+					// 	Cell: (cellProps: CellProps<FuturesTrade>) => (
+					// 		<Price>
+					// 			{cellProps.row.original.isOpen
+					// 				? '--'
+					// 				: formatCurrency(Synths.sUSD, cellProps.value, {
+					// 						sign: '$',
+					// 				  })}
+					// 		</Price>
+					// 	),
+					// 	width: 100,
+					// 	sortable: true,
+					// },
 					{
 						Header: (
-							<StyledTableHeader>{t('futures.market.user.trades.table.final')}</StyledTableHeader>
+							<StyledTableHeader>{t('futures.market.user.trades.table.pnl')}</StyledTableHeader>
 						),
-						accessor: 'exitPrice',
+						accessor: 'pnl',
 						sortType: 'basic',
-						Cell: (cellProps: CellProps<PositionHistory>) => (
-							<Price>
-								{cellProps.row.original.isOpen
-									? '--'
-									: formatCurrency(Synths.sUSD, cellProps.value, {
-											sign: '$',
-									  })}
-							</Price>
-						),
+						Cell: (cellProps: CellProps<FuturesTrade>) =>
+							cellProps.row.original.size.gt(wei(0)) ? (
+								<PNL negative={cellProps.value.lt(wei(0))}>
+									{formatCurrency(Synths.sUSD, cellProps.value, {
+										sign: '$',
+									})}
+								</PNL>
+							) : (
+								<PNL normal={true}>--</PNL>
+							),
```

Comment:
Some issue with the logic here. Closed positions have a size of 0, but with log some profit or loss